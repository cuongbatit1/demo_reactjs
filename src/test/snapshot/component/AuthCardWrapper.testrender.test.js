import { render, screen } from "@testing-library/react";
import AuthCardWrapper from "../../../component/AuthCardWrapper";

test("test renders AuthCardWrapper snapshot", () => {
  const { asFragment } = render(<AuthCardWrapper />);
  expect(asFragment()).toMatchSnapshot();
});
