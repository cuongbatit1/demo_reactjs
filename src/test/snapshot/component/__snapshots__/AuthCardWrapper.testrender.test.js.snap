// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders AuthCardWrapper snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-vlo4ox-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiBox-root css-1arlb8v"
    />
  </div>
</DocumentFragment>
`;
